app/views/admin/users/show/profile/actions/_delete.html.erb

```diff
@@ -2,13 +2,13 @@
   <% if current_user.super_admin? %>
     <%= form_for(@user, url: full_delete_admin_user_path(@user), html: { class: "flex flex-col gap-4", method: :post, onsubmit: "return confirm('Are you sure? This action is irreversible.')", id: nil }) do |f| %>
       <p>This action is irreversible.</p>
-      <p>Once deleted, all data for <%= @user.name %> will be removed from our database.</p>
+      <p>Once deleted, all data of <%= @user.name %> will be removed from our database.</p>
```

Comment:
I'm sure this is correct grammar but it feels off to me... "data relating to"? IDK feel free to ignore this vague musing, my grasp of English isn't amazing 😂

Reply:
I felt the same way, but was privately informed that it should be "of," which is why I opened this PR in the first place! I actually heavily prefer "data relating to" but... 🤷‍♀️  😄

Reply:
@amywtlin are you okay with me changing this to read "data relating to" instead of "data of"? I think that "data relating to" flows much nicer. How do you feel about this?

Reply:
**data relating to** sounds good to me ✨ Thank you both for catching things that feel off!